python/setup.py.in

```diff
@@ -9,36 +9,49 @@ import platform
 from setuptools import setup
 from setuptools.extension import Extension
 
-# These paths will be configured by CMake
+# These variables will be substituted in by CMake during configuration.
 PROJECT_SOURCE_DIR = "@PROJECT_SOURCE_DIR@"
 PROJECT_BINARY_DIR = "@PROJECT_BINARY_DIR@"
+SMT_SWITCH_LIB_TYPE = "@SMT_SWITCH_LIB_TYPE@"
+BITWUZLA_LDFLAGS = "@BITWUZLA_LDFLAGS@".split(";")
 
 # Get the platform-specific library extension
 if sys.platform == "darwin":
     lib_ext = ".dylib"
+    static_lib_ext = ".a"
 elif sys.platform == "win32":
     lib_ext = ".dll"
+    static_lib_ext = ".lib"
 else:  # linux
     lib_ext = ".so"
+    static_lib_ext = ".a"
 
 def find_solvers():
     """Find all compiled solvers based on built libraries in the build directory."""
     all_solvers = set()
     build_dir = Path(PROJECT_BINARY_DIR)
 
     # First verify the base library exists
-    base_lib_name = f"libsmt-switch{lib_ext}"
-    base_lib_path = build_dir / base_lib_name
-    if not base_lib_path.exists():
-        raise FileNotFoundError(f"Could not find base library: {base_lib_name} in {build_dir}")
+    base_lib = "libsmt-switch"
+    for ext in [lib_ext, static_lib_ext]:
+        base_lib_name = base_lib + ext
+        base_lib_path = build_dir / base_lib_name
+        if base_lib_path.exists():
+            break
+    else:
```

Comment:
The whitespace looks odd here. Is this just GitHub getting something wrong or are the if/else not lined up correctly?

Reply:
That `else` is part of the `for` loop; see my comment above.